packages/ckit/src/tx-builders/ExchangeSudtForCkbBuilder.ts

```diff
@@ -0,0 +1,203 @@
+import { Address, Cell, HexNumber } from '@ckb-lumos/base';
+import { CkbTypeScript } from '@ckitjs/base';
+import {
+  Amount,
+  AmountUnit,
+  Builder,
+  Cell as PwCell,
+  RawTransaction,
+  Transaction,
+  cellOccupiedBytes,
+  WitnessArgs,
+} from '@lay2/pw-core';
+import { BigNumber } from 'bignumber.js';
+import { Pw } from '../helpers/pw';
+import { CkitProvider } from '../providers';
+import { AbstractPwSenderBuilder } from './pw/AbstractPwSenderBuilder';
+
+const minExchangeProviderCkb = new Amount('441', AmountUnit.ckb); // 379 + 61 + 1
```

Comment:
the `minExchangeProviderCkb` cannot hard-code here, it should be `ckbAmountForRecipient + minimalCellCapacity(exchangeProviderCell)`

Reply:
this.minExchangeProviderCkb() has calculated it, but the calculation method is wrong. I'll modify it.